Pod/Extensions/SSO/SKYAuthContainer+SSO.m

```diff
@@ -161,4 +164,30 @@ - (NSURL *)sso_genCallbackURL:(NSString *)scheme
     return [[NSURL alloc] initWithScheme:scheme host:host path:@"/auth_handler"];
 }
 
+#pragma mark - Custom Token
+
+- (void)loginWithCustomToken:(NSString *)customToken
+           completionHandler:(SKYContainerUserOperationActionCompletion)completionHandler
+{
+    SKYLoginCustomTokenOperation *op =
+        [SKYLoginCustomTokenOperation operationWithCustomToken:customToken];
+    op.loginCompletionBlock = ^(SKYRecord *user, SKYAccessToken *accessToken, NSError *error) {
+        if (error) {
+            if (completionHandler) {
+                completionHandler(user, error);
```

Comment:
I think we also need that here, running in main thread. after updating this, we can merge. :P